src/content-scripts/components/GenerateAIDescription/DescriptionGeneratorButton.ts

```diff
@@ -0,0 +1,55 @@
+import { createHtmlElement } from "../../../utils/createHtmlElement";
+import openSaucedLogoIcon from "../../../assets/opensauced-icon.svg";
+import { getPullRequestAPIURL } from "../../../utils/urlMatchers";
+import { getPRDiff, getPRCommitMessages } from "../../../utils/aiprdescription/fetchGithubAPIData";
+import { generateDescription } from "../../../utils/aiprdescription/openai";
+import { GITHUB_PR_COMMENT_TEXT_AREA_SELECTOR } from "../../../constants";
+import { insertAtCursorFromStream } from "../../../utils/aiprdescription/cursorPositionInsert";
+import { getAIDescriptionConfig } from "../../../utils/aiprdescription/descriptionconfig";
+
+export const DescriptionGeneratorButton = () => {
+  const descriptionGeneratorButton = createHtmlElement("a", {
+    innerHTML: `<span id="ai-description-gen" class="toolbar-item btn-octicon">
+    <img class="octicon octicon-heading" height="16px" width="16px" id="ai-description-button-logo" src=${chrome.runtime.getURL(openSaucedLogoIcon)}>
+    </span>
+    <tool-tip for="ai-description-gen">Generate PR description</tool-tip>`,
+    onclick: handleSubmit,
+
+  });
+
+  return descriptionGeneratorButton;
+};
+
+
+const handleSubmit = async () => {
+  try {
+    const logo = document.getElementById("ai-description-button-logo");
+    if (!logo) {
+      return alert("Logo not found!");
```

Comment:
The User does not need to know about this, again a console.warn might make more sense. Even if we send this alert to the user, it does not explain anything.

Reply:
I don't remember why I set that to alert. I'll fix it.